mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -718,9 +723,9 @@ public static MailboxPath getPathLike(MailboxQuery mailboxQuery, MailboxSession
         return mailboxMapper.findNonPersonalMailboxes(session.getUser().getUserName(), Right.Lookup).stream();
     }
 
-    private boolean isReadable(MailboxSession session, Mailbox mailbox) throws MailboxException {
+    private boolean isReadable(MailboxSession session, Mailbox mailbox, Right right) throws MailboxException {
         return (isSameUser(session, mailbox) && isUserNamespace(mailbox))
-                || hasRight(mailbox, Right.Lookup, session);
+                || hasRight(mailbox, right, session);
```

Comment:
isReadable is not the good name anymore I think because it can be readable to check up other thinks !
Why not ``hasRightForUser``